auth/service.go

```diff
@@ -603,13 +603,17 @@ func (svc service) CanAccessGroup(ctx context.Context, token, groupID string) er
 		return err
 	}
 
+	if len(op.Orgs) == 0 {
```

Comment:
The length check is going to done in the for loop, there is no need to do it here. Better move the error at the end of the method